base/sparse/cholmod.jl

```diff
@@ -1325,4 +1325,7 @@ function ishermitian(A::Sparse{Complex{Float64}})
     end
 end
 
+(*){Ti}(A::Symmetric{Float64,SparseMatrixCSC{Float64,Ti}}, B::SparseMatrixCSC{Float64,Ti}) = sparse(Sparse(A)*Sparse(B))
```

Comment:
I don't this will be a huge problem. If you want to do work in this file you'd have to figure out what `Sparse` and `Factor` mean inside the module. Furthermore, this method definition should go away as soon as we provide the functionality in pure Julia which will provide efficiency and element type generality.